lib/buildScripts.coffee

```diff
@@ -0,0 +1,28 @@
+spawn = require('child_process').spawn
+
+module.exports =
+
+    build: ->
+        # start Hint
+        atom.notifications.addInfo 'Build started', dismissable: true, detail: 'Please dont Start ArmA until the Build Progress is Done'
```

Comment:
Shorter is better here as well as some capitalization is off.
`Please don't start ArmA until the build is completed.`